devito/types/dense.py

```diff
@@ -1327,15 +1327,15 @@ def forward(self):
         i = int(self.time_order / 2) if self.time_order >= 2 else 1
         _t = self.dimensions[self._time_position]
 
-        return self.subs({_t: _t + i * _t.spacing})
+        return self._subs(_t, _t + i * _t.spacing)
```

Comment:
out of curiosity: was this change necessary for functionality or for speed?

Reply:
ah ok, it's because now subs accepts a single pair `old,new` as two separate args